reporter/src/funTest/kotlin/reporters/NoticeReporterTest.kt

```diff
@@ -23,48 +23,31 @@ import com.here.ort.model.OrtResult
 import com.here.ort.model.config.CopyrightGarbage
 import com.here.ort.model.readValue
 import com.here.ort.reporter.DefaultResolutionProvider
-import com.here.ort.utils.safeDeleteRecursively
 
-import io.kotlintest.TestCase
-import io.kotlintest.TestResult
 import io.kotlintest.matchers.string.shouldContain
 import io.kotlintest.matchers.string.shouldNotContain
 import io.kotlintest.shouldBe
 import io.kotlintest.specs.WordSpec
 
+import java.io.ByteArrayOutputStream
 import java.io.File
 
 class NoticeReporterTest : WordSpec() {
     companion object {
         private fun readOrtResult(file: String) = File(file).readValue<OrtResult>()
     }
 
-    private lateinit var tempDir: File
-
-    override fun beforeTest(testCase: TestCase) {
-        super.beforeTest(testCase)
-        tempDir = createTempDir()
-    }
-
-    override fun afterTest(testCase: TestCase, result: TestResult) {
-        tempDir.safeDeleteRecursively(force = true)
-        super.afterTest(testCase, result)
-    }
-
-    private fun generateReport(ortResult: OrtResult,
-                               copyrightGarbage: CopyrightGarbage = CopyrightGarbage(),
-                               postProcessingScript: String? = null
-    ): String {
-        NoticeReporter().generateReport(
-                ortResult,
-                DefaultResolutionProvider(),
-                copyrightGarbage,
-                tempDir,
-                postProcessingScript
-        )
-
-        return File(tempDir, "NOTICE").readText()
-    }
+    private fun generateReport(ortResult: OrtResult, copyrightGarbage: CopyrightGarbage = CopyrightGarbage(),
+                               postProcessingScript: String? = null) =
```

Comment:
Please put the arguments on separate lines:
https://kotlinlang.org/docs/reference/coding-conventions.html#function-formatting

Reply:
Done, except that my IDE automatically indents 8 spaces instead of 4 spaces for some reason...

Reply:
Same here, haven't noticed that before.